Sources/ExyteChat/Theme/ChatTheme.swift

```diff
@@ -209,6 +209,25 @@ public struct ChatTheme {
     }
 
     public struct Images {
+      
+        public struct Background {
+            let portraitBackgroundLight: Image
+            let portraitBackgroundDark: Image
+            let landscapeBackgroundLight: Image
+            let landscapeBackgroundDark: Image
+
+            public init(
```

Comment:
in order to use this functionality, you need to set all images, this makes sense as without setting both landscape and portrait images, it won't look correct in landscape mode, the user could set all images to be the same if they wanted